tests/testthat/test-pkgdown.R

```diff
@@ -0,0 +1,39 @@
+context("pkgdown")
+
+get_exported <- function() {
+  if ('pkgload' %in% installed.packages() && pkgload::is_dev_package('shiny')) {
```

Comment:
The `pkgload::` is causing a WARNING in R CMD check. I don't think we should add it as a Suggested package, so we'll need a different way of detecting whether the package was loaded via pkgload/devtools.